airflow-core/docs/migrations-ref.rst

```diff
@@ -39,7 +39,9 @@ Here's the list of all the Database Migrations that are executed via when you ru
 +-------------------------+------------------+-------------------+--------------------------------------------------------------+
 | Revision ID             | Revises ID       | Airflow Version   | Description                                                  |
 +=========================+==================+===================+==============================================================+
-| ``0242ac120002`` (head) | ``dfee8bd5d574`` | ``3.1.0``         | Change the Deadline column in the Deadline table from        |
+| ``1b612ec87098`` (head) | ``0242ac120002`` | ``3.1.0``         | make bundle_name not nullable                                |
```

Comment:
We should make the description a full sentence to match the style. (This needs to be done in the migration file and synced to the doc.)